airflow/www/utils.py

```diff
@@ -682,6 +684,12 @@ def get_attr_renderer():
     }
 
 
+def get_trimmed_field(query: Query, model: Model, column_name: str) -> tuple[Query, ColumnElement]:
```

Comment:
`get_trimmed_field_setup_query` seems to be a more accurate name 🤔 
```suggestion
def get_trimmed_field_setup_query(query: Query, model: Model, column_name: str) -> tuple[Query, ColumnElement]:
```